GettingStarted.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Please, name all html file using lowerCase. Rename this file to `gettingStarted.html`